### DISCUSSION TITLE
Setting AutoCAD version - Newbie

### DISCUSSION THREAD

**Discussion by CWCWOOD** [haplokuon/netDxf#438] [General]
At this point I am only creating DXF files using netDXF, great job on your project! I am trying to figure out how to set the AutoCAD support version or if this is even necessary? I am using...

DxfVersion dxfVersion = DxfVersion.AutoCad14;

...but when I check the version (before loading it) as in the example, the same version is always displayed of 2000...

dxfVersion = DxfDocument.CheckDxfFileVersion(file);

...it was recommended to me that I produce files that support AutoCAD 2014.

Thank you.

**Comment by CWCWOOD** [comment]
I found the answer...

doc.DrawingVariables.AcadVer = DxfVersion.AutoCad2000;

...sorry for the rookie mistake.

**Comment by DavidWishengrad** [comment]
I recommend setting that to 2010. 

You can also take advantage of GPT4 and Githib copilot for coding and explanation of the many fine samples in the source code.

Keep in mind that Acad has fearures and functions that are not documented and supported in dxf format, such as dynamic blocks and 3D solids, for example. Possibly arrays too (I saw something that had an array not come back correctly, but I have not had the time to look into it further yet).

have fun and good luck.
